plugin.googledrive/addon.xml

```diff
@@ -36,7 +36,20 @@ Play all your media from Google Drive including Videos, Music and Pictures (incl
 		<source>https://github.com/cguZZman/plugin.googledrive</source>
 		<forum>https://github.com/cguZZman/plugin.googledrive/issues</forum>
 		<website>https://addons.kodi.tv/show/plugin.googledrive</website>
+		<assets>
+			<icon>icon.png</icon>
+			<fanart>fanart.jpg</fanart>
+		</assets>
 		<news>
+v1.3.3 released Feb 14, 2020:
```

Comment:
please keep this  only for the last version in the next update.

Reply:
Noted. Will do.